### DISCUSSION TITLE
autoSlide mode follows verticals despite grid navigation

### DISCUSSION THREAD

**Discussion by jsaponara** [hakimel/reveal.js#3143] [Q&A] [answered]
Is there a way to skip the verticals in autoSlide mode?  Using grid navigation, moving through the slides with the right arrow key skips the verticals, as desired, but the autoSlide mode follows each vertical.  I see this behavior in versions 4.2.1 and 4.0.2.  Thanks!

Using settings:
navigationMode: 'grid',
autoSlide: 4000,
autoSlideMethod: Reveal.navigateRight,
transition: "slide",
controls: true,
progress: true,
history: true,

If this is intended, what code should I tweak to get the behavior I want?

**Comment by hakimel** [comment, marked as answer]
You can fix this by using
```js
autoSlideMethod: () => Reveal.navigateRight()
```

As of reveal.js 4.0 `Reveal.navigateRight` isn't available before reveal.js is initialized so it ends up being undefined in your original example. There's no need to specify a `navigationMode` for this to work. I'll update the outdated example in the docs so no one else runs into this.

**Reply by jsaponara** [reply]
Awesome, thanks!

**Reply by mex511511mno** [reply]
وش فيه انا متهكر ليش
